models/arrays_varname/arrays_varname_spec.json

```diff
@@ -1,18 +1,5 @@
 {
   "name": "Arrays test model (where the spec file contains Vensim variable names)",
-  "inputVarNames": [
-    "inputA[A1]",
-    "inputA[A2]",
-    "inputA[A3]"
-  ],
-  "outputVarNames": [
-    "Time",
-    "a[A1]",
-    "a[A2]",
-    "a[A3]",
-    "f[A1,B1]",
-    "f[A1,B2]",
-    "f[A1,B3]",
-    "h"
-  ]
+  "inputVarNames": ["inputA[A1]", "inputA[A2]", "inputA[A3]"],
+  "outputVarNames": ["Time", "a[A1]", "a[A2]", "a[A3]", "f[A1,B1]", "f[A1,B2]", "f[A1,B3]", "h"]
```

Comment:
Ugh, this kind of change is my main gripe with Prettier, and there's no way around it, so I'll just accept it like I've accepted so many other Prettier changes 😄